import-export-cli/cmd/getAPILogging.go

```diff
@@ -75,9 +75,12 @@ func init() {
 
 	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingAPIId, "api-id", "i",
 		"", "API ID")
+	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingTenantId, "tenant-id", "",
+		"", "TENANT ID")
 	getApiLoggingCmd.Flags().StringVarP(&getApiLoggingEnvironment, "environment", "e",
 		"", "Environment of the APIs which the API loggers should be displayed")
 	getApiLoggingCmd.Flags().StringVarP(&getAPILoggingCmdFormat, "format", "", "", "Pretty-print API loggers "+
 		"using Go Templates. Use \"{{ jsonPretty . }}\" to list all fields")
+	_ = getApiLoggingCmd.MarkFlagRequired("tenant-id")
```

Comment:
Without mandating the tenant-id flag, shall we use carbon.super if the falg is not provided?

Reply:
We can check whether the value is empty and if so set carbon.super as the tenant-id value when calling the rest APIs

Reply:
tenant-id is not mandatory now. default value is set to carbon.super